.github/workflows/release.yml

```diff
@@ -31,28 +31,23 @@ jobs:
     name: Release
     runs-on: ubuntu-latest
     steps:
-      - uses: webfactory/ssh-agent@v0.4.1
+      - uses: webfactory/ssh-agent@v0.7.0
         with:
           ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
       - name: Clone repository
         run: git clone git@github.com:Khan/wonder-blocks.git .
 
-      - name: Setup Node.js 16.x
-        uses: actions/setup-node@master
+      - name: Install & cache node_modules
+        uses: Khan/actions@shared-node-cache-v0
         with:
-          # This is our target Node version.
```

Comment:
**note:** This comment wasn't in Wonder Stuff so it got removed when I pasted in the new step. I can add it back if folks feel strongly about it.